src/audio.js

```diff
@@ -0,0 +1,38 @@
+import * as THREE from "three";
+import songUrl from "./sounds/bg1.mp3";
+import songUrl1 from "./sounds/tile.mp3";
+
+//adding background sound
+ export const addAudioListenerToCamera = (camera) => {
+  camera.add(createAudioListener());
```

Comment:
This causes an error because the `camera` here is not a `PerspectiveCamera`
Should be `camera.camera.add(createAudioListener())`